docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -1168,6 +1182,10 @@ public URI getDockerUri() {
         return dockerUri;
     }
 
+    public boolean isDockerInsideDockerResolution() {
```

Comment:
This Object is passed around (config is not) and providing this here is useful to understand the client context.